src/agent/src/mount.rs

```diff
@@ -444,16 +444,19 @@ fn mount_storage(logger: &Logger, storage: &Storage) -> Result<()> {
     let options_vec = options_vec.iter().map(String::as_str).collect();
     let (flags, options) = parse_mount_flags_and_options(options_vec);
 
+    let source = Path::new(&storage.source);
+    let mount_point = Path::new(&storage.mount_point);
+
```

Comment:
`Path::new` returns an `&Path` not a bare `Path` (which is unsized)....

Reply:
I've now resolved all those `Path` issues afaik.